alternator/executor.cc

```diff
@@ -84,6 +84,21 @@ std::string make_jsonable::to_json() const {
     return rjson::print(_value);
 }
 
+json::json_return_type make_streamed(rjson::value&& value) {
+    // CMH. json::json_return_type uses std::function, not noncopyable_function. 
+    // Need to make a copyable version of value. Gah.
+    auto rs = make_shared<rjson::value>(std::move(value));
+    std::function<future<>(output_stream<char>&&)> func = [rs](output_stream<char>&& os) mutable -> future<> {
+        auto los = std::move(os);
+        auto lrs = std::move(rs);
+        co_await rjson::print(*lrs, los);
+        co_await los.flush();
```

Comment:
My own implementation of this function didn't have flush() before the close() and it worked, so perhaps flush() isn't necessary? But I guess it can't hurt.

Reply:
In the current case, no, but why not?